node_modules/col-assets/lib/rest.js

```diff
@@ -70,3 +70,29 @@ Collabosphere.apiRouter.post('/assets', function(req, res) {
     return res.status(400).send('Unrecognized asset type');
   }
 });
+
+/*!
+ * Create a new comment on an asset
+ */
+Collabosphere.apiRouter.post('/assets/:assetId/comments', function(req, res) {
+  AssetsAPI.createComment(req.ctx, req.params.assetId, req.body.body, req.body.parent, function(err, comment) {
+    if (err) {
+      return res.status(err.code).send(err.msg);
+    }
+
+    return res.status(201).send(comment);
+  });
+});
+
+/*!
+ * Edit a comment on an asset
+ */
+Collabosphere.apiRouter.post('/assets/:assetId/comments/:commentId', function(req, res) {
+  AssetsAPI.editComment(req.ctx, req.params.assetId, req.params.commentId, req.body.body, function(err, comment) {
+    if (err) {
+      return res.status(err.code).send(err.msg);
+    }
+
+    return res.status(200).send(comment);
+  });
+});
```

Comment:
Deleting comments is currently to be determined, which is why I left it out of this PR.